Increase project limit default to 300 (#2665)
chef/automate: components/automate-chef-io/content/docs/iam-v2-guide.md

#### Configuring Project Limit

> By default, Chef Automate limits you to 300 projects. You can increase the project limit using the command line.

Reviewer: Global search for "six projects" turns up one more occurrence in iam-v2-overview.md.

> By default, Chef Automate is currently limited to 300 projects. If you would like to increase that limit, you will need to do so using the Chef Automate CLI:

Reviewer 2: So this line was originally reworked to decrease the use of passive tense and simplify the sentence structure. I recommend just changing the number to `300`.
```suggestion
By default, Chef Automate limits you to 300 projects. You can increase the project limit using the command line.
```

Author: I strictly swapped the numbers-- not sure how this old text took over. Maybe a bad rebase 🤷‍♀

Reviewer 2: Totally understand. I was mildly suspicious the text change was a computer conspiracy, not actually by a user. It's so frustrating when that happens!
